app/components/SongCard/index.js

```diff
@@ -0,0 +1,121 @@
+/**
+ *
+ * MusicCard
+ *
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Card, Typography, Button } from 'antd';
+import { PlayCircleTwoTone, StopTwoTone } from '@ant-design/icons';
+import { colors } from '@app/themes';
+import handleMusic, { actions } from '@app/utils/audioControl.js';
+import { translate } from '../IntlGlobalProvider/index';
+import { T } from '../T/index';
+
+const { Paragraph } = Typography;
+
+const Container = styled(Card)`
+  && {
+    width: ${(props) => (props.width ? props.width : 25)}em;
+    height: ${(props) => (props.height ? props.height : 32)}em;
+    background-color: ${colors.songCardBg};
+    display: block;
+    border-radius: 2em;
+    box-shadow: inset -6px -6px 12px rgba(0, 0, 0, 0.8), inset 6px 6px 12px rgba(255, 255, 255, 0.4);
+    overflow-x: scroll;
+    &::-webkit-scrollbar {
+      display: none;
+    }
+  }
+`;
+
+const IconsContainer = styled.div`
+  display: flex;
+  align-items: center;
+  width: 100%;
+  justify-content: space-between;
+`;
+
+const CommonIconStyle = {
```

Comment:
camelCase for Objects.

styled-components are Pascal-case because they are used as Components

Reply:
Changed accordingly